tests/file_system_cache_test.py

```diff
@@ -56,5 +56,48 @@ def test_empty_key_get(self):
      with self.assertRaisesRegex(ValueError , r"key cannot be empty"):
        cache.get("")
 
+  def test_size_of_directory(self):
+    with tempfile.TemporaryDirectory() as tmpdir:
+      cache = FileSystemCache(tmpdir)
+      cache.put("foo", b"bar")
+      self.assertEqual(cache._get_cache_directory_size(), 3)
+
+  def test_size_of_empty_directory(self):
+    with tempfile.TemporaryDirectory() as tmpdir:
+      cache = FileSystemCache(tmpdir)
+      self.assertEqual(cache._get_cache_directory_size(), 0)
+
+  def test_size_of_existing_directory(self):
+    with tempfile.TemporaryDirectory() as tmpdir:
+      cache1 = FileSystemCache(tmpdir)
+      cache1.put("foo", b"bar")
+      del cache1
+      cache2 = FileSystemCache(tmpdir)
+      self.assertEqual(cache2._get_cache_directory_size(), 3)
+
+  def test_cache_is_full(self):
+    with tempfile.TemporaryDirectory() as tmpdir:
+      cache = FileSystemCache(tmpdir, max_cache_size_bytes = 5)
```

Comment:
Same as above with the keyword argument `=`, should be `max_cache_size_bytes=5` with no spaces